opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -1473,9 +1473,9 @@ void CpGrid::addLgrsUpdateLeafView(const std::vector<std::array<int,3>>& cells_p
     // Map to relate boundary patch faces with their children refined/new-born ones. {0,oldFaceIdx} -> {level,{newFaceIdx0, ...}}
     std::map<std::array<int,2>, std::tuple<int, std::vector<int>>> old_to_new_boundaryPatchFaces;
     //
-    // For level0, attach children to each parent cell. For no parents, entry {-1, {}} representing {no level, {no children}}
+    // For level0, attach children to each parent cell. For no parents, entry {invalidIdx, {}} representing {no level, {no children}}
```

Comment:
On second thought:
The first item is the level, right. Maybe it is better to keep using -1 here?